lib/helpers.js

```diff
@@ -848,35 +848,38 @@ function toAvdLocaleArgs (language, country) {
  * @returns {?string} The full path to the folder or `null` if the folder cannot be found
  */
 async function getAndroidPrefsRoot () {
-  // https://gerrit.pixelexperience.org/plugins/gitiles/development/+/5c11852110eeb03dc5a69111354b383f98d15336/tools/androidprefs/src/com/android/prefs/AndroidLocation.java
-  let location = null;
-  if (system.isWindows()) {
-    let localAppData = process.env.LOCALAPPDATA;
-    if (!localAppData) {
-      localAppData = process.env.USERPROFILE;
-      if (localAppData) {
-        localAppData = path.resolve(localAppData, 'Local Settings', 'Application Data');
-      }
-    }
-    if (localAppData) {
-      location = path.resolve(localAppData, 'Android');
-    }
-  } else {
-    const home = process.env.HOME;
-    if (home) {
-      location = path.resolve(home, '.android');
-    }
+  let location = process.env.ANDROID_EMULATOR_HOME;
+  if (await dirExists(location)) {
+    return location;
   }
 
-  if (location && await fs.exists(location)) {
-    if (!(await fs.stat(location)).isDirectory()) {
-      log.debug(`Android config root '${location}' is not a directory`);
-      return null;
-    }
-    return location;
+  if (location) {
+    log.warn(`The value of $ANDROID_EMULATOR_HOME '${location}' is not an existing directory`);
```

Comment:
I would rather explicitly mention `The value of ANDROID_EMULATOR_HOME environment variable`, because under windows env variables look like `%VAR%`